src/EA.Weee.Web/Areas/AATFReturn/Controllers/AatfTaskListController.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Api.Client;
+    using EA.Weee.Requests.AatfReturn;
+    using EA.Weee.Requests.Organisations;
+    using EA.Weee.Web.Areas.AatfReturn.ViewModels;
+    using EA.Weee.Web.Constant;
+    using EA.Weee.Web.Controllers.Base;
+    using EA.Weee.Web.Infrastructure;
+    using EA.Weee.Web.Services;
+    using EA.Weee.Web.Services.Caching;
+
+    public class AatfTaskListController : ExternalSiteController
+    {
+        private readonly Func<IWeeeClient> apiClient;
+        private readonly BreadcrumbService breadcrumb;
+        private readonly IWeeeCache cache;
+        private readonly IMapper mapper;
+
+        public AatfTaskListController(Func<IWeeeClient> apiClient, BreadcrumbService breadcrumb, IWeeeCache cache, IMapper mapper)
+        {
+            this.apiClient = apiClient;
+            this.breadcrumb = breadcrumb;
+            this.cache = cache;
+            this.mapper = mapper;
+        }
+
+        [HttpGet]
+        public virtual async Task<ActionResult> Index(Guid organisationId, Guid returnId)
+        {
+            var viewModel = new ReturnViewModel();
+            using (var client = apiClient())
+            {
+                var @return = await client.SendAsync(User.GetAccessToken(), new GetReturn(returnId));
```

Comment:
Ideally here we don't want to do 2 requests to get the data we need... Same is making a change that returns the operator details with the return. When we come back to do more work on this screen we will do a bit of refactoring- so will leave this at the moment.